INSTALLATION.md

```diff
@@ -0,0 +1,19 @@
+# Installation
+
+## Prerequisites
+
+- Node.js LTS Dubnium or newer
+- [Yarn](https://yarnpkg.com/en/)
+- Google Chrome
+
+## Authentication
+
+Follow this guide to set up [Authentication](AUTHENTICATION.md)
+
+## Install Dependencies
+
+Open terminal window in each of the component directories eq-author, eq-author-api and eq-publisher then run
+```
+yarn
+```
+to install all dependencies (running `yarn` on it's own is the same as running `yarn install`)
```

Comment:
Do we need to specify this?  We could just link to the `yarn` documentation? 

https://yarnpkg.com/en/docs/installing-dependencies